hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/AbstractReplicatedMapOperation.java

```diff
@@ -28,7 +29,7 @@
 
 import static com.hazelcast.replicatedmap.impl.ReplicatedMapService.INVOCATION_TRY_COUNT;
 
-public abstract class AbstractReplicatedMapOperation extends AbstractSerializableOperation {
+public abstract class AbstractReplicatedMapOperation extends AbstractNamedSerializableOperation implements NamedOperation {
```

Comment:
The `implements NamedOperation` is unnecessary.

Reply:
Fixed.